### DISCUSSION TITLE
LazyVim: Disable notifications from specific plugin

### DISCUSSION THREAD

**Discussion by itzcodex24** [folke/noice.nvim#672] [Q&A]
How can I disable notifications initiated by one plugin "presence.nvim"

![image](https://github.com/folke/noice.nvim/assets/70026038/06bea112-db35-443a-aad6-9cd34e9ee1ed)

This doesn't seem to work

**Comment by ilias777** [comment]
Try this:

```lua
{
    filter = {
        event = 'notify',
        kind = 'info',
        any = {
            { find = 'available' }, -- for No information available message
            { find = 'presence' }, -- maybe this works for presence.nvim
        },
    },
    opts = { skip = true },
},
```

I don't know what messages comes from presence.nvim and what kind they are, but maybe you can add in the `any` table a second `find` field as you see above. If the `kind` is not `info` for the presence.nvim, then this won't work.  For the *No information available* is this the correct way.
